doc/gitlinker.txt

```diff
@@ -88,7 +88,7 @@ looking for is `require"gitlinker".get_buf_range_url(mode, user_opts)` where:
 
 >
   vim.api.nvim_set_keymap('n', '<leader>gb', '<cmd>lua require"gitlinker".get_buf_range_url("n", {action_callback = require"gitlinker.actions".open_in_browser})<cr>', {silent = true})
-  vim.api.nvim_set_keymap('v', '<leader>gb', '<cmd>lua require"gitlinker".get_buf_range_url("v", {action_callback = require"gitlinker.actions".open_in_browser})<cr>'))
+  vim.api.nvim_set_keymap('v', '<leader>gb', '<cmd>lua require"gitlinker".get_buf_range_url("v", {action_callback = require"gitlinker.actions".open_in_browser})<cr>', {silent = true})
```

Comment:
I think this suggestion is more personal and probably related to the warning from https://github.com/ruifm/gitlinker.nvim/issues/43, so I'm not sure it should be part of the "default" mapping advice.

Reply:
I snuck it in here because I didn't feel like it warranted its own PR 🙈. I had some issues copy pasting this particular snippet into my config because it had an extra parentheses at the end, and then `vim.api.nvim_set_keymap` expects 4 arguments, so I figured it could be like the other examples (`{silent = true}`)